Makefile

```
@@ -14,7 +14,7 @@ hello: check_python
 virtualenv: check_env
 	@if test -s "$$VIRTUAL_ENV"; then echo -e "\nRun 'deactivate' first\n"; false; fi
 	if test -e .venv; then rm -rf .venv/; fi
-	python3.12 -m venv .venv
+	python3.13 -m venv .venv
```

```suggestion
	python -m venv .venv
```

In a separate commit. If you find any invocations of `python3`, please also replace them with `python`.